### ISSUE TITLE
Send hit 10s API timeout (but succeeded onchain)

### ISSUE THREAD

**Issue by dcposch** [daimo-eth/daimo#1138]
## Summary

Observed this afternoon. Likely related to the issue Ale saw:

```
2024-06-10T23:48:58.422Z log [ACTION] useSendAsync - 10540ms: idle > error Error sending transaction
2024-06-10T23:48:58.422Z log [ACTION] useSendAsync - error, total time 10540ms
2024-06-10T23:48:58.423Z log [SEND] rendering SendButton 0.12
2024-06-10T23:48:58.429Z ERR [SEND] error: TRPCClientError: Aborted
2024-06-10T23:48:59.202Z log [TRPC] timeout after 10000ms: https://api.daimo.xyz/chain/8453/getAccountHistory?batch=1&input=%7B%220%22%3A%7B%22address%22%3A%220x4D350d99364634e07B01a9986662787DD3755F0A%22%2C%22inviteCode%22%3A%22dc-aws%22%2C%22sinceBlockNum%22%3A15636375%7D%7D
2024-06-10T23:48:59.205Z ERR [SYNC] FAILED dcposch TRPCClientError: Aborted
    at construct (native)
[...]
2024-06-10T23:49:00.387Z log [TRPC] GET validateMemo 502 in 853ms
2024-06-10T23:49:00.388Z log [TRPC] GET getAccountHistory 502 in 980ms
2024-06-10T23:49:00.389Z ERR [SYNC] FAILED dcposch TRPCClientError: JSON Parse error: Unexpected character: <
    at construct (native)
```

**Comment by dcposch** [comment]
## Investigation

Not necessarily related--`getAccountHistory` should never block on routes--but the Uniswap route finder is extremely slow.

```
2024-06-10 16:48:58.360 [daimo_api] DaimoApiCluster-production api-task-production:17 stdout [UNISWAP] fetched swap for 0x4D350d99364634e07B01a9986662787DD3755F0A: 50000000000000000 GRG 0x09188484e1Ab980DAeF53a9755241D759C5B7d60 in 40538ms
```

**Comment by dcposch** [comment]
False alarm, bad logging. Fixed in #1158
